proposed/http-middleware/middleware-meta.md

```diff
@@ -270,75 +276,37 @@ additional information._
 
 [promises]: https://promisesaplus.com/
 
-### 5.2 Delegate Design
-
-The `DelegateInterface` defines a single method that accepts a request and
-returns a response. The delegate interface must be implemented by any middleware
-dispatcher that uses middleware implementing `MiddlewareInterface`.
-
-#### Why the term "delegate"?
-
-The term "delegate" means something designated to act for or represent another.
-In terms of middleware design, a delegate is called upon by middleware when the
-middleware is unable to process the request itself; the delegate then processes
-the request _for the original middleware_ in order to return a response.
```

Comment:
The point of the meta document is to be able to explain the logic behind the choices. If we decide to drop the `delegate` name, we should explain why. (because if we can't that means that we maybe shouldn't)